oda_api/data_products.py

```diff
@@ -240,9 +276,9 @@ def _check_dict(self,_kw):
 
 
     @classmethod
-    def from_fits_hdu(cls,hdu,name=''):
+    def from_fits_hdu(cls,hdu,name=None):
 
-        if name=='':
+        if not name:
```

Comment:
I think PEP8 recommends to always use `is` operator for checking against `None`

Reply:
Here I wanted to account for both None and empty string (compatibility again, but not sure if it's really needed)

Reply:
then  I'd put:

```python
if name is not None or name == '':
```